content/html/concepts/tags/terms/q/q.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '<q>'
+Description: 'Used to represent a brief inline quotation.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Attributes'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<q>` tag is used to represent a brief inline quotation.
+It accepts a `cite` attribute which will link to a source that provides further context for the quotation.
+
+## Syntax
+
```

Comment:
```suggestion

The `<q>` tag requires an opening and closing tag wrapped around a line of text. The browser will automatically enclose the text in quotation marks:

```

Reply:
Hey @Dusch4593 - apologies for missing this one. Before I commit and to ensure I understand correctly, should the final result look like this: 

![image](https://user-images.githubusercontent.com/39813213/156812728-aa648561-5726-4dff-b829-0d187641e0ac.png)

Reply:
Yep, that's right! 👍